### PR TITLE
Don't cache row heights in recordings event list

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Addresses bug captured in https://github.com/PostHog/posthog/issues/6483#issuecomment-975777421. This comes out of a call with @rcmarron where we decided to standardize row heights for performance reasons.

- [ ] Changes all event row heights in the events list to a predictable two lines. This makes for easier rendering math and virtualization that speeds up scrolling significantly and squashes some highlighting bugs.

Screen Shot 2021-11-22 at 1 18 46 PM

- [ ] Since important information can be truncated, any rows where the description was cut off can be hovered over to reveal the long form in position.

https://user-images.githubusercontent.com/13460330/142936818-1416fb7d-3777-4f55-8393-5f8fab67a2b5.mov

- [ ] Adds title prop to descriptions

## How did you test this code?

Observe less buggier highlighting and scrolling on recordings with lots of events.

### CODE REVIEW

**Comment by rcmarron** [comment]
Thoughts on changing the description text on autocapture events, so that we're more likely to see the interesting part. We could do 'Clicked "Internal Metrics"' if the element has text and otherwise use the element type (e.g. 'Clicked span')

File: frontend/src/scenes/session-recordings/player/styles.scss [hunk 1]
```diff
@@ -42,9 +42,7 @@
         .player-sidebar {
             display: flex;
             flex-direction: column;
-            width: 30%;
-            min-width: 300px;
-            max-width: 400px;
+            width: 350px;
```
Comment (alexkim205) [inline frontend/src/scenes/session-recordings/player/styles.scss:45]: Static width = less rendering that react-virtualized has to do.

File: frontend/src/scenes/session-recordings/player/PlayerEvents.scss [hunk 2]
```diff
@@ -164,14 +163,31 @@
                         padding-right: $recording_spacing / 2;
                     }
 
-                    .event-item-content-subtitle {
-                        display: -webkit-box;
-                        -webkit-line-clamp: 2;
-                        -webkit-box-orient: vertical;
+                    .event-item-content-timestamp {
```
Comment (rcmarron) [inline frontend/src/scenes/session-recordings/player/PlayerEvents.scss:166]: Nit: looks like some leftovers

**Review by rcmarron — APPROVED** [review]
Looks great! Soo much smoother!

**Pull request merged**